### PR TITLE
MAIN Add message notification components

### PR DESCRIPTION
**Pull request by herman-rogers**
image

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
+import "react-toastify/dist/ReactToastify.min.css";
```
Comment (paschalidi) [inline src/index.js]: does this really need to be here?

Comment (herman-rogers) [reply src/index.js]: as in? Put it elsewhere?

File: src/Message/Message.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      progressClassName: "progress-bar"
+    });
+  },
+  error(text, options = {}) {
+    return toast.error(
+      <MessageContent
+        icon="failed-filled-16"
+        text={text}
+        color="#e94119"
+        size={18}
```
Comment (paschalidi) [inline src/Message/Message.js]: why would this be size?

Comment (paschalidi) [inline src/Message/Message.js]: can we also rename this to fontSize?

File: src/Message/Message.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  color: PropTypes.string,
+  size: PropTypes.number
+};
+
+MessageContent.defaultProps = {
+  color: "#04d4bf",
+  size: 22
+};
+
+export function MessageContainer() {
```
Comment (paschalidi) [inline src/Message/Message.js]: why not `export default`?

**Comment by paschalidi** [comment]
Do you mind if we name this Notification(s)?
`Message` is too generic and we might need that in the future for some sort of "Message". 
I believe Notifications is more what this popup thingy is.

File: src/Message/Message.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  return (
+    <Styles>
+      <ToastContainer />
+    </Styles>
+  );
+}
+
+export const message = {
+  ...toast,
+  info(text, options = {}) {
```
Comment (paschalidi) [inline src/Message/Message.js]: can you explain, maybe with a story how this `options` can be used?

File: src/index.js (hunk 1 above)
Comment (paschalidi) [reply src/index.js]: yes maybe we can have it in the file `src/Message/Message.js`? i have the feeling it is more part of the component than part of the exports.

File: src/Message/Message.js (hunk 2 above)
Comment (herman-rogers) [reply src/Message/Message.js]: it's too big otherwise

Comment (paschalidi) [reply src/Message/Message.js]: Cant all then be 18px?

File: src/Message/Message.js (hunk 3 above)
Comment (herman-rogers) [reply src/Message/Message.js]: because there's multiple exports inside of here.

File: src/Message/Message.js (hunk 4 above)
Comment (herman-rogers) [reply src/Message/Message.js]: There's a documentation story that explains it

File: src/Message/Message.js (hunk 2 above)
Comment (herman-rogers) [reply src/Message/Message.js]: So they're not all uniform in size. If they are all the same size some of them will be larger than others. The Error icon is particularly bigger than the rest.

Comment (paschalidi) [reply src/Message/Message.js]: I see. Thanks

**Review by paschalidi — APPROVED** [review]

**Pull request merged**